### DISCUSSION TITLE
download link?

### DISCUSSION THREAD

**Discussion by csoyuncu** [theAsmodai/metamod-r#80] [Q&A] [answered]
where is the download link of metamod-r ?

**Comment by hajimura** [comment]
Nowhere, because it isn't ready for correct working on the server, need bugfixes

**Comment by theAsmodai** [comment, marked as answer]
https://github.com/theAsmodai/metamod-r/releases/tag/1.3.66

**Comment by stamepicmorg** [comment]
maby close? :)
